### PR TITLE
Feature/allow passing parameters to tasks

### PR DESCRIPTION
**Pull request by sebastien-p**
See #3.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,15 +38,29 @@ module.exports = function(userOpts) {
...
+			if (stat.isFile() && opts.extensions.indexOf(extension) === -1) {
 				return;
 			}
 
-			var taskname = path.basename(filename, path.extname(filename));
+			var taskname = path.basename(filename, extension);
 			var taskinfo = require(file);
 
+			if (opts.params !== undefined) {
+				if (!Array.isArray(taskInfo)) {
```
Comment (rummik) [inline index.js]: Probably want to `%s/taskInfo/taskinfo/g` on this.  That's the only issue I'm seeing.

Comment (sebastien-p) [inline index.js]: I’m sorry but I don’t understand what you mean.

Comment (rummik) [inline index.js]: `taskInfo` isn't defined, but `taskinfo` is (note the capitalization) -- sorry <.<;

Comment (sebastien-p) [inline index.js]: Oh I see, sorry, done.

**Comment by rummik** [comment]
Sweet.  I'll review and merge as soon as I'm at a computer.

**Comment by sebastien-p** [comment]
In this case @rummik should increment the package version number to _0.9.0_ and `$ npm publish` the new version. That would take roughly 30 seconds. Please @rummik do this :)

**Pull request merged**